hibernate-testing/src/main/java/org/hibernate/testing/junit4/BaseUnitTestCase.java

```diff
@@ -35,7 +36,8 @@
 @RunWith( CustomRunner.class )
 public abstract class BaseUnitTestCase {
 
-	static {
+	@BeforeClassOnce
+	public void clearSchemas() {
 		DatabaseCleaner.clearSchemas();
```

Comment:
@beikov it's a bad idea to do this sort of thing in a static initialized, because errors get swallowed and turned into NCDFEs.

Reply:
I know, sorry you had issues with this :(

Reply:
Unfortunately this change wasn't enough to eliminate the NCDFEs in all cases. :-(

We need to figure out something better.